src/components/MenuLeft/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import Item from '../ListItem';
+import messages from './messages';
+import Modal from './Modal';
+import A from './A';
+import DropDownNavBar from './DropdownNavBar';
+
+function MenuLeft() {
+  return (
+    <Modal className="inset-0">
+      <div className="absolute inset-y-0 left-0 w-1/5 bg-indigo-500 py-12">
+        <div className="flex justify-center">
+          <A href="#">
```

Comment:
mấy cái thẻ cơ bản như thẻ a, img thì viết thẳng luôn không cần viết thành component đâu 
Ví dụ như 1 cái này cần viết component nè 
---------
Image. tức là thẻ img nó nhận 1 cái src, mà lỡ như 1 user A nào đó nó ko có avatar --> lúc này sẽ show avatar dèault ( có nghĩa là ở component này có liên quan đến việc xử lý logic + sẽ xài ở nhiều nơi ) ==> cần thiết viết component 
Còn thẻ a, thì tất nhiên là xài nhiều nơi nhưng nó ko có xử lý logic gì thì ko cần tách ra 1 component làm chi

Reply:
Ok, nhưng bin đã styled cái pseudo :hover rồi nên cái component này sẽ cần

Reply:
ko liên quan đến cái style, chủ yếu là cách mình tổ chức á, 1 file mà component với ccais the html nó lộn xộn lên thì k tốt